db/models.py

```diff
@@ -0,0 +1,25 @@
+from sqlalchemy import Integer, Column, String, Date, ForeignKey
+from sqlalchemy.orm import relationship
+
+from db.engine import Base
```

Comment:
The import statement should be `from db.database import Base` instead of `from db.engine import Base` to correctly import the `Base` object.